php/Factory.php

```diff
@@ -15,64 +12,23 @@
 class Factory
 {
     /**
-     * @return ContainerBuilder
-     */
-    public function createDIContainer($configFileName, $configDir, $additionalParameters = [])
-    {
-        $container = new ContainerBuilder;
-
-        $loader = new YamlFileLoader($container, new FileLocator($configDir));
-
-        $loader->load($configFileName);
-
-        foreach ($additionalParameters as $name => $value) {
-            $container->setParameter($name, $value);
-        }
-
-        $container->compile();
-
-        return $container;
-    }
-
-    /**
-     * @param string $implementation absolute namespace path to a concrete logger
-     * @param array $parameters parameters to logger's constructor
+     * @param string $path the log file path
      * @return \Psr\Log\LoggerInterface
      */
-    public function createLogger($implementation, $parameters = [])
+    public static function createLogger($path, $is_null = false)
```

Comment:
In fact, the Factory class is just some util class which is just for convenient. I don't see any necessity to make an instance of it. In fact, any method written in Factory class could be move to the main.php, and will never be reused in other place.